src/App/component/ComparisonPage.jsx

```diff
@@ -72,18 +72,34 @@ function ComparisonPage(prop) {
   };
 
   const projectId = localStorage.getItem("projectId")
-  const jwtToken = localStorage.getItem("jwtToken")
+//   const jwtToken = localStorage.getItem("jwtToken")
 
-  useEffect(() => {
-    Axios.get(`http://localhost:9100/pvs-api/project/1/${projectId}`,
-      { headers: { "Authorization": `${jwtToken}` } })
-      .then((response) => {
-        setCurrentProject(response.data)
+  const sendPVSBackendRequest = async (method, url, jwt) => {
+    const baseURL = 'http://localhost:9100/pvs-api';
+    const requestConfig = {
+      baseURL,
+      url,
+      method,
+      headers: {...(jwt && {'Authorization': jwt})}
+    };
+    return (await Axios.request(requestConfig))?.data;
+  };
+
+  const loadInitialProjectInfo = () => {
+    sendPVSBackendRequest('GET', `/project/1/${projectId}`)
+      .then((responseData) => {
+        if (responseData) {
+          setCurrentProject(responseData)
+        }
       })
       .catch((e) => {
         alert(e)
         console.error(e)
       })
+  }
+
+  useEffect(() => {
+    loadInitialProjectInfo() // <<-- Now it has a name, so we can easily know what is its purpose ...
```

Comment:
Please remove the tutorial comment, it is just for explaining why to use it, not means it should be involved into  our real code base.